compute/admin_guide/compliance/detect_secrets.adoc

```diff
@@ -1,12 +1,16 @@
-== Detect secrets
+== Detect Secrets
 
 Prisma Cloud can detect sensitive information that is improperly secured inside images and containers.
 Scans can detect embedded passwords, login tokens, and other types of secrets.
 To detect improperly secured secrets, add the following checks to your xref:../compliance/manage_compliance.adoc#[compliance policy].
 
 
-[.section]
-=== Compliance check ID 424
+=== Defender Secret Detection
+
+Your deployed defenders detect sensitive information inside images and containers.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using the `secrets` keyword to see the impacted images and containers.
```

Comment:
filtering with "secrets" won't give the the relevant checks, a proper way to look would be to search for the relevant checks you listed below by IDs